src/components/Tabs/docs/index.story.tsx

```diff
@@ -35,5 +58,65 @@ export default {
     variant: Object.keys(VARIANT).map(key => VARIANT[key]),
     items: exampleItems,
   },
-  examples: <Examples />,
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          importExample("import { Tabs } from 'wix-ui-tpa/Tabs';"),
+
+          divider(),
+
+          title('Examples'),
+
+          code(<Examples />),
+        ],
+      }),
+
+      ...[
+        { title: 'API', sections: [api()] },
+        { title: 'Style API', sections: [styleApi({ css })] },
+        { title: 'TestKit', sections: [testkit()] },
+        { title: 'Playground', sections: [playground()] },
+        {
+          title: 'Settings Panel',
+          sections: [
+            settingsPanel({
+              title: 'Tabs Panel',
+              example: <TabsExtendedExample items={items} />,
```

Comment:
I would aim to have this created automatically, look very much like `playground`.
so that the same meta data that you scrape from the `StyleApi` will be available for this panel as well.
it will be an auto-generated opinionated panel